request-management-api/request_api/services/documentservice.py

```diff
@@ -77,7 +77,10 @@ def createministrydocumentversion(self, ministryrequestid, documentid, documents
         document = FOIMinistryRequestDocument.getdocument(documentid)
         if document:
             FOIMinistryRequestDocument.deActivateministrydocumentsversion(documentid, document['version']+1, userid)
-            return FOIMinistryRequestDocument.createdocumentversion(ministryrequestid, version, self.__copydocumentproperties(document,documentschema,document['version']), userid)          
+            createdat = document ['created _at'] # store this because __copydocumentproperties function below overwrites it
```

Comment:
Apart from the obvious issue in the other files above, the __copydocumentproperties function was overwriting the document created_at with None. This seems intentional so I didn't want to touch the __copydocumentproperties function, just did a simple workaround here. Please just confirm that __copydocumentproperties is functioning as intended